FlowCrypt/Functionality/Services/Account Server Services/EnterpriseServerApi.swift

```diff
@@ -101,14 +101,21 @@ class EnterpriseServerApi: EnterpriseServerApiType {
             throw EnterpriseServerApiError.emailFormat
         }
 
-        let response: ClientConfigurationResponse = try await performRequest(
-            email: email,
-            url: "/api/v1/client-configuration?domain=\(userDomain)",
-            method: .get,
-            withAuthorization: false
-        )
+        do {
+            let response: ClientConfigurationResponse = try await performRequest(
+                email: email,
+                url: "/api/v1/client-configuration?domain=\(userDomain)",
+                method: .get,
+                withAuthorization: false
+            )
+
+            return response.clientConfiguration
+        } catch EnterpriseServerApiError.noActiveFesUrl {
```

Comment:
Maybe it's better to not throw error in `getActiveFesUrl` for `publicEmailProviderDomains`

Reply:
I think we can leave `throw` there, but improve it's handling, so users will be able to re-login after startup error

Reply:
Let's leave the throw for now and improve handling as Roman says. We could re-visit it in the future.